java/src/test/java/org/whispersystems/libsignal/devices/DeviceConsistencyTest.java

```diff
@@ -0,0 +1,76 @@
+// TODO: this test was failing even before our changes, figure out why
```

Comment:
As noted, this test was failing even before our changes.

Reply:
Interesting!